arm-compute/2016-09-30/swagger/containerService.json

```diff
@@ -291,7 +291,8 @@
               "enum": [
                 "Swarm",
                 "DCOS",
-                "Custom"
+                "Custom",
+                "Kubernetes"
```

Comment:
@brendandburns - I agree that this is an additive change w.r.t API. However, the API-version remains the same. I think the SDKs (.net, java) sdk have already shipped and this would be a breaking change for customers. The api-version has not changed. So if the customer changed or created something via portal and tries to get the same thing via .net sdk then the sdk will throw an exception as ContainerServiceOrchestratorProfile would not know that Kubernetes is a valid value.